### PR TITLE
Update sidebar selected database or collection 

### PR DESCRIPTION
**Pull request by ziftinpeki**
I've updated sidebar styling for selected DB and selected collection and added extra control method for "setDatabase" and "setCollection" methods before calling them.

### CODE REVIEW

File: src/components/connections.vue [hunk 1]
```diff
@@ -31,33 +48,47 @@ export default {
...
+      button(type='button', @click='doConnect') Connect
   .box
-     h1 Databases
-     .database(v-for="database in databases" @click="setDatabase(database.name)")
-       p {{database.name}}
+    h1 Databases
+    .box-item(
+      v-for='database in databases',
+      @click='setSelectedDBorCollection("db", database.name)',
+      :class='{ selected: database.name === databaseName ? true : false }'
```
Comment (dashersw) [inline src/components/connections.vue:65]: `database.name == databaseName` is enough here, you don't need a ternary. the same for the collections below.

File: src/components/connections.vue [hunk 2]
```diff
@@ -10,18 +10,35 @@ export default {
     return {
       name: connectionName,
       connectionString,
-      isNewConnectionFormVisible: false
+      isNewConnectionFormVisible: false,
+      isConnected: null
```
Comment (dashersw) [inline src/components/connections.vue:14]: what do we do with `isConnected` here?

File: src/components/connections.vue [hunk 3]
```diff
@@ -10,18 +10,35 @@ export default {
...
     }
   },
   methods: {
     ...mapActions(['connect', 'setDatabase', 'setCollection']),
     async doConnect() {
       await this.connect(this.connectionString)
+      this.isConnected = true
       this.$router.push('/connection')
+    },
+    setSelectedDBorCollection(sourceType, sourceName) {
```
Comment (dashersw) [inline src/components/connections.vue:24]: i believe you don't need this method here. simply returning false in the store in both `setDatabase` and `setCollection` if the names are the same should solve this issue.

**Comment by ziftinpeki** [comment]
my other PR (#32 ) solves all of them we don't need this PR

**Pull request closed without merging**